ml-agents/mlagents/trainers/sac/policy.py

```diff
@@ -58,6 +58,7 @@ def __init__(
 
         with self.graph.as_default():
             # Create pretrainer if needed
+            self.bc_module: Optional[BCModule] = None
```

Comment:
Should we use the same logic for PPO?

Reply:
It wasn't complaining about PPO (probably because no type hints there) but makes sense to do it there too.

Reply:
Done now.

Reply:
:+1: Thanks for adding the type hints to PPO too.